cg/server/invoices/templates/invoices/layout.html

```diff
@@ -12,7 +12,7 @@
   <!-- Bootstrap CSS -->
   <link
     rel="stylesheet"
-    href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css"
+    href="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
     integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb"
```

Comment:
OK, the integrity hash might have to change tho! It seems modern browsers will calculate this to make sure the referenced file is indeed correct.

Reply:
See https://getbootstrap.com/docs/4.0/getting-started/download/#bootstrapcdn

Reply:
Tested this and the layout was "down graded" to look like some web sight in the 90s.

Reply:
If you test with the latest changes, does it still look downgraded?